deployments/deploy-ssh.sh

```diff
@@ -80,8 +80,8 @@ if [ -n "${TORPEDO_SSH_KEY}" ]; then
     TORPEDO_SSH_KEY_MOUNT="{ \"name\": \"ssh-key-volume\", \"mountPath\": \"/home/torpedo/\" }"
 fi
 
-TESTRESULTS_VOLUME="{ \"name\": \"testresults\", \"hostPath\": { \"path\": \"/testresults/\", \"type\": \"DirectoryOrCreate\" } }"
-TESTRESULTS_MOUNT="{ \"name\": \"testresults\", \"mountPath\": \"/testresults/\" }"
+TESTRESULTS_VOLUME="{ \"name\": \"testresults\", \"hostPath\": { \"path\": \"/mnt/testresults/\", \"type\": \"DirectoryOrCreate\" } }"
+TESTRESULTS_MOUNT="{ \"name\": \"testresults\", \"mountPath\": \"/mnt/testresults/\" }"
```

Comment:
@nikolaypopov ,

Inside the pod, we can keep the mount path to just `/testresults` since our torpedo tests write JUnit xml files in `/testresults` as done [here](https://github.com/portworx/torpedo/blob/master/tests/basic/basic_test.go#L21)

we can just change the host directory path to `/mnt/testresults` if is allowed to create directory there by RHEL 8.